frappe/email/doctype/auto_email_report/auto_email_report.json

```diff
@@ -147,6 +147,7 @@
      "label": "Email Settings"
     },
     {
+     "description": "For multiple addresses, enter the address on different line. e.g. test@test.com \u23ce test1@test.com",
```

Comment:
The modified timestamp also needs to be updated, otherwise the schema will not be updated on migrate.